### ISSUE TITLE
Duping bug found with Sanguimancy

### ISSUE THREAD

**Issue by NateG763** [breakinblocks/nutsandboltstorqued#162]
I play on a private server and we found one user abusing a duping bug using the Altar diviner from Sanguimancy. Someone posted the issue to Saunguimancy back in the end of February but it hasn't seemed to be addressed/fixed.

https://github.com/Tombenpotter/Sanguimancy/issues/122

**Comment by Saereth** [comment]
yep im gonna disable it in the next update then till its fixed, (updat comes out tomorrow or the next day)
